cypress/e2e/login_management/cloudLoginAndLogout.feature

```diff
@@ -10,22 +10,28 @@ Feature: Login Management - Login of users managed by SVS
     Scenario Outline: User makes a direct login with username and password
 
         Given I am logged in as a '<student>' at '<namespace>'
-        Then I log out
+        Then I see my initials '<student_initials>' in the dashboard
```

Comment:
I think, if we can create three scenarios for each users. In that case if any user fails for any reason then it will not skip for other user which is the case now.

Reply:
I created an own example for each user and reduced the scenario to one login and logout.